src/QsCompiler/CompilationManager/PerformanceTracking.cs

```diff
@@ -64,6 +65,12 @@ public enum Task
             /// </summary>
             RewriteSteps,
 
+            /// <summary>
+            /// Task for a specific rewrite step.
+            /// These tasks should be accompanied with details of which rewrite step it is specific to.
+            /// </summary>
+            SingleRewriteStep,
```

Comment:
I would add this to the end of the enum such that the current integer value of the Tasks that are already part of the enum.

Reply:
done